packages/opentelemetry-exporter-zipkin/src/transform.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+import { ReadableSpan } from '@opentelemetry/basic-tracer';
+import * as zipkinTypes from './types';
+
+const MICROS_PER_MILLI = 1000;
+
+// TODO: what should be the mapping?
+const STATUS_CODE = 'ot.status_code';
+const STATUS_DESCRIPTION = 'ot.status_description';
+
+const ZIPKIN_SPAN_KIND_MAPPING = {
+  [types.SpanKind.CLIENT]: zipkinTypes.SpanKind.CLIENT,
+  [types.SpanKind.SERVER]: zipkinTypes.SpanKind.SERVER,
+  [types.SpanKind.CONSUMER]: zipkinTypes.SpanKind.CONSUMER,
+  [types.SpanKind.PRODUCER]: zipkinTypes.SpanKind.PRODUCER,
+  // Zipkin doesnt have type INTERNAL
+  [types.SpanKind.INTERNAL]: zipkinTypes.SpanKind.CLIENT,
```

Comment:
Hmm, not sure about this. Just add `@todo` here and will revisit again.

Reply:
I think we should skip the span bcoz `INTERNAL` indicates that the span is used internally.

Reply:
Are you sure?
`SpanKind.Internal MUST be assumed as a default.` - spec

This would mean that we would skip on every span by default, except someone defines SpanKind. OpenCensus also sends it as a client, so would make migration more challenging.

Reply:
Hmm, I checked out Zipkin docs again and sounds like we can just leave it `undefined` and let JSON.stringify to remove it.
```
kind: When present, kind clarifies timestamp, duration and remoteEndpoint. When
                    absent, the span is local or incomplete. 
```